fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -517,6 +517,8 @@ public static MaterializedViewRewriteMode parse(String str) {
 
     public static final String CBO_PUSHDOWN_TOPN_LIMIT = "cbo_push_down_topn_limit";
 
+    public static final String ADAPTIVE_CHOOSE_EXECUTE_INSTANCES_MODE = "adaptive_choose_execute_instances_mode";
```

Comment:
can remove the prefix `adaptive_`? the default mode is locality?
to reduce size, can replace `instances` with `nodes`?

Reply:
Add a deault mode `locality`.  Instance is inheriting the old semantics, it may be easier for users to understand. With `adaptive_`, I think it may more clear.